app/assets/stylesheets/notice.scss

```diff
@@ -0,0 +1,17 @@
+.notice {
+  background-color: #D2D6DB; // smoke-20
```

Comment:
You can technically access onboarding steps having applied one of the dark themes. Probably for that reason would be good sticking to our existing approach with setting colors, which is:

```scss
// reference: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/_mixins.scss#L9
// usage example: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/articles.scss#L73
@mixin themeable($property, $cssVariable, $fallback)
```

Reply:
I am hypothesizing that people who are onboarding have not set up a theme yet, which is I went with this fixed value. What do you think?

Reply:
It will be very rare edge case BUT you can go straight to `https://dev.to/onboarding` while being logged in. Which means you could have set alternative theme in the meantime. Again, won't happen very often but worth adding it in for sake of making things bulletproof :D

Reply:
Resolving this since I'm removing this file :)